packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -68,8 +70,11 @@ export default {
     },
 
     data () {
+        uid += 1;
```

Comment:
Does this correctly increment when adding multiple components to the page?

Reply:
Hey @DamianMullins!  It does yeah. I've tested it with the three links in registration and each one has a different ID

Reply:
🎉